src/effects/native/phasereffect.cpp

```diff
@@ -173,7 +192,7 @@ void PhaserEffect::processChannel(const ChannelHandle& handle,
         right = processSample(right, oldInRight, oldOutRight, filterCoefRight, stages);
 
         // Computing output combining the original and processed sample
-        pOutput[i] = pInput[i] * (1.0 - 0.5 * depth) + left * depth * 0.5;
-        pOutput[i + 1] = pInput[i + 1] * (1.0 - 0.5 * depth) + right * depth * 0.5;
+        pOutput[i] = pInput[i] + left * depth * 0.5;
+        pOutput[i + 1] = pInput[i + 1] + right * depth * 0.5;
```

Comment:
Unlike Reverb, the Phaser can be used as an insert effect. Since it has no delay component, you can turn it into a kind of insert effect by using the dry wet knob only. After this change, this is not possible any-more.

How about issue the "not loud enough" problem by a special scale for depth: 
-> left half, new scale: up to 100 % dry + 50 % phaser 
-> right half: up to 0% dry + 100 % phaser